ganga/GangaCore/Lib/Condor/Condor.py

```diff
@@ -797,4 +797,38 @@ def timedetails(self):
     updateMonitoringInformation = \
         staticmethod(updateMonitoringInformation)
 
+    def peek(self, filename=None, command=None):
+        job = self.getJobObject()
+
+        
+                    
+
+        idElementList = job.backend.id.split("#")
+        if 3 == len(idElementList):
+            if idElementList[1].find(".") != -1:
+                peekCommand = f"condor_tail -f -name {idElementList[0]} {idElementList[1]}"
+            else:
+                peekCommand = f"condor_tail -f -name {idElementList[0]} {idElementList[1]}"
+        else:
+            peekCommand = f"condor_tail -f {idElementList[0]}"
+
+        status, output = subprocess.getstatusoutput(peekCommand)
+
+
+        if (status != 0):
+            logger.warning \
+                ("Return code '%s' peek job '%s' - Condor id '%s'" %
+                 (str(status), job.id, job.backend.id))
+            logger.warning("Tried command: '%s'" % peekCommand)
+            logger.warning("Command output: '%s'" % output)
+        else:
+            logger.info(output)
+
+        peekStatus = True
+        if job.subjobs:
+            peekStatus= False
+            logger.error("Master Job does not have a peek status.")
+
```

Comment:
Should this not be up at the top at line 803 instead? Code also seems a bit inconsistent in if for a failure returns